src/jobs/js_build_job.yml

```diff
@@ -0,0 +1,29 @@
+description: >
+  Build application
+parameters:
+  app_type:
+    type: enum
+    enum: [ 'legacy', 'nextjs' ]
+    default: nextjs
+  build_env:
+    type: enum
+    enum: ['beta', 'prod']
+    default: beta
+  resource_class:
+    type: string
+    default: large
+executor: js
+resource_class: << parameters.resource_class >>
+steps:
+  - checkout
+  - attach_workspace:
+      at: .
+  - run:
+      name: Delete .env
```

Comment:
Nice. Totally agree. As an information, .env are going to be .gitignored before the end of March. This is on my platter. This is related to the new "Secret Scanning" feature in GitHub and enforced by the Security Team at SMG level (Mosti, etc.) Look here as an example: https://github.com/ricardo-ch/marketplace-frontend-app/security/secret-scanning. 
My idea is to generate a NOT-COMMITTED local only .env file from isopod.yml and kubernetes secrets, using kubernetes API and yaml parsing. I can explain it to you if you want.